crates/ethrpc/src/block_stream/mod.rs

```diff
@@ -161,7 +161,7 @@ pub async fn current_block_stream(
 
             tracing::info!(number=%block.number, hash=?block.hash, "noticed a new block");
             if sender.send(block).is_err() {
-                tracing::debug!("exiting polling loop");
+                tracing::info!("exiting polling loop");
```

Comment:
The more I think about the more I think that this should be an error. It is not recoverable and catastrophic for the pod. It might even make sense to panic here because it's so bad. 🤔